src/Framework/MSBuildEventSource.cs

```diff
@@ -0,0 +1,364 @@
+﻿using System.Collections.Generic;
+using System.Diagnostics.Tracing;
+
+namespace Microsoft.Build.Eventing
+{
+
+    //
+    // This captures information of how various key methods of building with MSBuild ran.
+    //
+    // /OnlyProviders=*Microsoft-Build
```

Comment:
Admittedly, it's more important without the documentation. It could be helpful if you know how to run an eventsource from the command line in general but don't know how to run this particular one. Given that it's not a surprising name, and there's documentation, I don't think it's important anymore.